### ISSUE TITLE
[BUG] [MK3.5] Hotend fan speed - "N/A" despite self test passing

### ISSUE THREAD

**Issue by arekm** [prusa3d/Prusa-Firmware-Buddy#3746]
**Printer type** - MK3.5
**Printer firmware version** - 5.2.0
**Original or Custom firmware** - Original

**Describe the bug**

**How to reproduce**
Go to "Sensor info" menu and notice there is "N/A" at "Hotend Fan".

**Expected behavior**
Just like Print Fan - "0 % / stopped" if it's stopped or the real speed if it is not.

Note that fan speed test passes just fine and speed of both fans is visible in rpms when fan self test is being made.

![mk3 5-hotend-fan-na](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/50869/90b55872-07f7-44f2-a764-ed3e8994d56c)
![mk3 5-hotend-selftest](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/50869/e302b2d7-def3-4728-a7c7-b81d20a1d6ab)

(heater test fails because I'm using Revo and 5.2.0 firmware is not ready for it)

**Comment by NicholasRakita** [comment]
We're looking into this now.

Internal ticket: BFW-5076

**Comment by arekm** [comment]
I see there is a fix

````
commit 8a2615d50bf39b684edaa6439d107ba954786932
Author: info@revskills.de <info@revskills.de>
Date:   Tue Feb 13 11:09:30 2024 +0100

    Fix missing board temperature display on MK3.5
````

**Comment by michalrudolf** [comment]
Fixed and scheduled for upcoming MK3.5 release

**Comment by danopernis** [comment]
Yes, this is actually the same issue as https://github.com/prusa3d/Prusa-Firmware-Buddy/issues/4088 which is fixed since v6.2.0-alpha1
